config/aws_citc.py

```diff
@@ -0,0 +1,191 @@
+# This is an example configuration file
+
+# Note that CPU autodetect currently does not work with this configuration file on AWS
+# In order to do CPU autodetection, two changes are needed:
+# 1. Remove all '--export=NONE'
```

Comment:
@casparvl Can we extend this comment to explain *why* this temporary change is needed to make the autodetect work?
Likewise for using `srun` as launcher